.github/workflows/claude.yml

```
@@ -133,9 +133,14 @@ jobs:
           github_token: ${{ secrets.GITHUB_TOKEN }}
           trigger_phrase: "@claude"
 
-  # 新規追加：自動コードレビュージョブ
+  # 制限付き自動コードレビュージョブ
   auto-review:
-    if: github.event_name == 'pull_request'
+    if: |
+      github.event_name == 'pull_request' && 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)